vendor/vendor.json

```diff
@@ -6,1135 +6,1143 @@
 			"checksumSHA1": "eruIVA8JnsB23rVKjETHvqJ0sj8=",
 			"origin": "github.com/elastic/beats/vendor/github.com/DataDog/zstd",
 			"path": "github.com/DataDog/zstd",
-			"revision": "5554677dc86b78f48b1ff2a214c661f5f78cbfe1",
-			"revisionTime": "2019-04-05T18:09:50Z"
+			"revision": "5f947e7a07261cbc10c2928e12c6de5a1830a624",
+			"revisionTime": "2019-04-11T13:09:32Z"
 		},
 		{
 			"checksumSHA1": "AzjRkOQtVBTwIw4RJLTygFhJs3s=",
 			"origin": "github.com/elastic/beats/vendor/github.com/Microsoft/go-winio",
 			"path": "github.com/Microsoft/go-winio",
-			"revision": "5554677dc86b78f48b1ff2a214c661f5f78cbfe1",
-			"revisionTime": "2019-04-05T18:09:50Z"
+			"revision": "5f947e7a07261cbc10c2928e12c6de5a1830a624",
+			"revisionTime": "2019-04-11T13:09:32Z"
 		},
 		{
 			"checksumSHA1": "Yn3uTky/UjksI76cPRAxXomzMKM=",
 			"origin": "github.com/elastic/beats/vendor/github.com/Shopify/sarama",
 			"path": "github.com/Shopify/sarama",
-			"revision": "5554677dc86b78f48b1ff2a214c661f5f78cbfe1",
-			"revisionTime": "2019-04-05T18:09:50Z"
+			"revision": "5f947e7a07261cbc10c2928e12c6de5a1830a624",
+			"revisionTime": "2019-04-11T13:09:32Z"
 		},
 		{
-			"checksumSHA1": "DYv6Q1+VfnUVxMwvk5IshAClLvw=",
+			"checksumSHA1": "YYMvxHp5ZvVURHxMQudv4yY3jJI=",
```

Comment:
This is a leftover now that `Sirupsen` has changed to `sirupsen`, see new entry at https://github.com/elastic/apm-server/pull/2098/files#diff-bd290170e2912d3e8694db1a151066e5R2046